{"name": "Запрос на авторизацию с неверным логином возвращает код 404", "status": "passed", "steps": [{"name": "Отправить POST запрос на эндпоинт /api/v1/courier/login с неверным логином", "status": "passed", "parameters": [{"name": "create_new_courier", "value": "['pnfrrzsfdp', 'zrevychfof', 'azekpzowjx']"}], "start": 1709698925338, "stop": 1709698925517}], "start": 1709698925338, "stop": 1709698925517, "uuid": "c6440502-6c15-4581-92c9-3449cec75886", "historyId": "e56e8320d20085a95d09f6cb663a48c7", "testCaseId": "e56e8320d20085a95d09f6cb663a48c7", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_with_wrong_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-TTE0GV0"}, {"name": "thread", "value": "25968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}